viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1186,22 +1285,113 @@ export class Cognite3DViewer {
     return this._models.filter(x => x.type === type);
   }
 
+  /** @private */
+  private moveCameraTargetTo(target: THREE.Vector3, duration?: number) {
+    if (this.isDisposed) {
+      return;
+    }
+
+    if (duration === 0) {
+      this.controls.setState(this.getCameraPosition(), target);
+      return;
+    }
+
+    const { camera, raycaster, _minDefaultAnimationDuration, _maxDefaultAnimationDuration } = this;
+
+    if (duration == null) {
+      const distance = target.distanceTo(this.controls.getState().target);
+      duration = distance * 125; // 125ms per unit distance
+      duration = Math.min(Math.max(duration, _minDefaultAnimationDuration), _maxDefaultAnimationDuration);
```

Comment:
Tested on different models and looks ok for me. Also, this value is based on distance to the camera, so, from my understanding should behave the same in all models because when they are imported in reveal they have the same scale.